build-logic/performance-testing/src/main/groovy/gradlebuild.performance-templates.gradle

```diff
@@ -395,7 +395,7 @@ performanceTest.registerTestProject("largeAndroidBuild", RemoteProject) {
 performanceTest.registerTestProject("santaTrackerAndroidBuild", RemoteProject) {
     remoteUri = 'https://github.com/gradle/santa-tracker-android.git'
     // Pinned from main branch
-    ref = 'c26e2b8aa5c34758934009f1d5b0334f7fc2db5a'
+    ref = 'a6dc397049c41d58bc5677b15aad934c57636b36'
```

Comment:
I see this is a commit from your branch. Can we merge that branch and reference a commit on `main`?

Reply:
Yes, that is what I'd do if we are fine with the changes on my branch with the corresponding effects.